src/display/editor/editor.js

```diff
@@ -1857,6 +1862,17 @@ class AnnotationEditor {
     return this._uiManager.direction === "ltr" ? [1, 0] : [0, 0];
   }
 
+  get commentButtonColor() {
+    if (!this.color) {
+      return null;
+    }
+    let [r, g, b] = AnnotationEditor._colorManager.convert(
+      this._uiManager.getNonHCMColor(this.color)
```

Comment:
Looking at the following screenshot, made with the preview viewer, the comment icons are hard to quite hard to see because the icon itself is white on a light background. Should the icon outline be changed too?

<img width="920" height="737" alt="Image" src="https://github.com/user-attachments/assets/fbb51ea0-6613-41db-965a-76f9ea198484" />